### PR TITLE
Update skypeforlinux-64.deb to 8.51.0.72

### PR DESCRIPTION
**Pull request by eos-shell-build**
<i>(This pull request was automatically generated by [endlessm/flatpak-external-data-checker](https://github.com/endlessm/flatpak-external-data-checker). Please contact @wjt if you have any questions or complaints.)</i>

### CODE REVIEW

**Comment by develop7** [comment]
@wjt please throw release notes update in (see README.md for the link)

File: com.skype.Client.json [hunk 1]
```diff
@@ -8,31 +8,24 @@
     "command": "skype",
     "separate-locales": false,
     "finish-args": [
-        /* X11 + XShm access */
```
Comment (develop7) [inline com.skype.Client.json]: @wjt I don't like removing these comments

Comment (wjt) [reply com.skype.Client.json]: The problem here is that comments are not valid JSON. This tool uses the standard Python `json` module to parse these files, but to work around the presence of comments like these, it round-trips the data through json-glib to strip comments and other non-standard constructs like multi-line strings.

**Comment by wjt** [comment]
> @wjt Is there way it could update URL _only_?

For YAML manifests, I found a parser `ruamel.yaml` which, if used carefully, does a reasonable job of preserving formatting. Would you be willing to convert this manifest to YAML?

For JSON, it's harder. As mentioned above, this file is not actually valid JSON. Comments aside, I was not able to find a JSON library for Python that can preserve formatting. The formatting used by this tool is deterministic, so if you accept this PR, future PRs will indeed only change the URL/sha/size fields.

I briefly experimented with the following approach:

- reformat the file, without any other changes, and commit the result
- apply the update, commit the result
- revert the first commit
- squash all three commits down to one commit

But one really quickly runs into conflicts. In this case, for example, I think the rewrapping of the `only-arches` value would cause a conflict. I decided it's not worth spending more effort on this.

If you can think of a way to implement comment- and layout-preserving changes to this JSON-with-comments, I'm all ears :-)

**Comment by wjt** [comment]
> @wjt please throw release notes update in (see README.md for the link)

I actually don't have direct access to @eos-shell-build's repos so I can't push these changes. The PR is created with `maintainer_can_modify=True` so I believe you should be able to push changes to @eos-shell-build's branch here.

**Comment by develop7** [comment]
> I believe you should be able to push changes

Uh, no, I'm but a humble contributor :)

File: com.skype.Client.json (hunk 1 above)
Comment (develop7) [reply com.skype.Client.json]: > The problem here is that comments are not valid JSON.

Oh. Totally missed that. Fair enough.

**Comment by develop7** [comment]
> Would you be willing to convert this manifest to YAML?

It was an option? Sure thing then.

**Comment by wjt** [comment]
> Uh, no, I'm but a humble contributor :)

Oh, sorry! We're in the same boat then :-)

**Pull request closed without merging**